<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1527402604735" stop="1527402604770">
  <name>test01</name>
  <labels/>
  <test-cases>
    <test-case start="1527402604735" status="failed" stop="1527402604767">
      <name>Test_01.test_001[1]</name>
      <failure>
        <message>AssertionError: assert 1 != 1</message>
        <stack-trace>self = &lt;test01.Test_01 object at 0x059FB4F0&gt;, a = 1

    @pytest.mark.parametrize("a",[1,2,3])
    def test_001(self,a):
&gt;       assert  a !=1
E       assert 1 != 1

script\test01.py:6: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="16468-MainThread"/>
        <label name="host" value="DESKTOP-KEMCE3K"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1527402604767" status="passed" stop="1527402604768">
      <name>Test_01.test_001[2]</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="16468-MainThread"/>
        <label name="host" value="DESKTOP-KEMCE3K"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1527402604769" status="passed" stop="1527402604770">
      <name>Test_01.test_001[3]</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="16468-MainThread"/>
        <label name="host" value="DESKTOP-KEMCE3K"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
